L10nSnapshotTests/L10nSuite1SnapshotTests.swift

```diff
@@ -35,16 +35,18 @@ class L10nSuite1SnapshotTests: L10nBaseSnapshotTests {
         sleep(3)
         waitForExistence(app.scrollViews.staticTexts["WelcomeCardTitleLabel"], timeout: 15)
         snapshot("Onboarding-1")
+        
```

Comment:
Oh looks like Swiftlint is complaining about this line:
https://github.com/mozilla-mobile/firefox-ios/pull/11794/checks?check_run_id=8226000705
```
🛑 1 out of 1 tests have failed!
/Users/vagrant/git/L10nSnapshotTests/L10nSuite1SnapshotTests.swift
Lines should not have trailing whitespace.

warning:
Line:38 

```

Reply:
![ohno](https://user-images.githubusercontent.com/1740517/188944948-498fccd1-b9c7-4ae4-a4cc-cda13a00d937.gif)
Line 38 (a blank line) begins with a tab.